tests/semantic/ignore_files.json

```diff
@@ -0,0 +1,542 @@
+    "solidity/test/libsolidity/semanticTests/fallback/fallback_or_receive.sol",
+    "solidity/test/libsolidity/semanticTests/fallback/fallback_override.sol",
+    "solidity/test/libsolidity/semanticTests/fallback/fallback_override2.sol",
+    "solidity/test/libsolidity/semanticTests/fallback/fallback_override_multi.sol",
+    "solidity/test/libsolidity/semanticTests/fallback/fallback_return_data.sol",
+    "solidity/test/libsolidity/semanticTests/fallback/inherited.sol",
+    "solidity/test/libsolidity/semanticTests/fallback/short_data_calls_fallback.sol",
+    "solidity/test/libsolidity/semanticTests/array/delete/delete_removes_bytes_data.sol",
+    "solidity/test/libsolidity/semanticTests/fallback/expressions/uncalled_address_transfer_send.sol",
+    "solidity/test/libsolidity/semanticTests/reverts/revert_return_area.sol",
+    "solidity/test/libsolidity/semanticTests/smoke/fallback.sol",
+
+
+    "// not implemented on starknet" ,
+    "solidity/test/libsolidity/semanticTests/reverts/invalid_instruction.sol",
+    "solidity/test/libsolidity/semanticTests/uninitializedFunctionPointer/invalidInConstructor",
+    "solidity/test/libsolidity/semanticTests/various/address_code.sol",
+    "solidity/test/libsolidity/semanticTests/various/address_code_complex.sol",
+    "solidity/test/libsolidity/semanticTests/various/code_access_content.sol",
+    "solidity/test/libsolidity/semanticTests/various/code_access_create.sol",
+    "solidity/test/libsolidity/semanticTests/various/code_access_padding.sol",
+    "solidity/test/libsolidity/semanticTests/various/code_access_runtime.sol",
+    "solidity/test/libsolidity/semanticTests/various/code_length.sol",
+    "solidity/test/libsolidity/semanticTests/various/code_length_contract_member.sol",
+    "solidity/test/libsolidity/semanticTests/various/codehash.sol",
+    "solidity/test/libsolidity/semanticTests/various/codehash_assembly.sol",
+    "solidity/test/libsolidity/semanticTests/various/codehash_assembly.sol",
+    "solidity/test/libsolidity/semanticTests/various/contract_binary_dependencies.sol",
+    "solidity/test/libsolidity/semanticTests/various/gasleft_decrease.sol",
+    "solidity/test/libsolidity/semanticTests/various/gasleft_shadow_resolution.sol",
+
+    "solidity/test/libsolidity/semanticTests/abiEncoderV2/cleanup/cleanup.sol",
+
+    "// overlfow (failed assert in cairo)",
+    "solidity/test/libsolidity/semanticTests/abiEncoderV2/cleanup/intx.sol",
+    "solidity/test/libsolidity/semanticTests/exponentiation/signed_base.sol",
+    "solidity/test/libsolidity/semanticTests/operators/shifts/bitwise_shifting_constantinople_combined.sol",
+    "solidity/test/libsolidity/semanticTests/operators/shifts/shift_overflow.sol",
+    "solidity/test/libsolidity/semanticTests/viaYul/exp_neg_overflow.sol",
+    "solidity/test/libsolidity/semanticTests/viaYul/exp_overflow.sol",
+
+    "// user defined value type (not propperly supported in nethersolc)",
```

Comment:
They should be now

Reply:
This branch is still using kudu, I am not sure if this branches kudu supports them?

Reply:
Hmm, makes sense then. It probably doesn't, kudu used to just throw when it encountered user-defined value types